### ISSUE TITLE
Feedback Email Entry Box Is too Passive

### ISSUE THREAD

**Issue by InsertCreativityHere** [icerpc/icerpc-docs#351]
When you open the "Leave Feedback" thing, it has a dialogue box to enter your email.
This box will let you enter pretty much anything in it (which is fine), but if you mouse over the box, you can see there's actually errors being reported. These errors are only visible upon mouse over and there is also no other suggestion something is wrong.

![image](https://github.com/icerpc/icerpc-docs/assets/25963603/13f4a3f6-1649-473f-be7f-9457cc6e5d6c)

We should either enforce the error (not let the user send, or put a little red symbol somewhere), or stop reporting it if we don't enforce it.
I lean towards the 2nd.

**Comment by InsertCreativityHere** [comment]
I think this is actually the browser's doing.
On Chrome the error text is different...

**Comment by ReeceHumphreys** [comment]
> I think this is actually the browser's doing.
> On Chrome the error text is different...

Yes this input type was set to email which hints to the browser what type of input to accept. We do not do any validatio on it otherwise we should probably do.
